tests/components/wemo/test_config_flow.py

```diff
@@ -0,0 +1,21 @@
+"""Tests for Wemo config flow."""
+
+from homeassistant import data_entry_flow
+from homeassistant.components.wemo.const import DOMAIN
+from homeassistant.config_entries import SOURCE_USER
+from homeassistant.core import HomeAssistant
+
+from tests.common import patch
+
+
+async def test_not_discovered(hass: HomeAssistant) -> None:
+    """Test setting up with no devices discovered."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={"source": SOURCE_USER},
+    )
+
+    with patch("pywemo.discover_devices", return_value=[]):
```

Comment:
Can you change the patch target only patch where the library is imported ?

Reply:
I think b709092 is what you were requesting. Please let me know if that isn't correct.

Reply:
You could probably patch homeassistant.components.wemo.config_flow.pywemo..... without changing the import

Reply:
Ah yes! Thank you for the clarification.